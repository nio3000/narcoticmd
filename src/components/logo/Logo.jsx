import React from 'react';
import LogoImage from './img/logo.svg';
const Logo = ( alt="Nacotic logo" )  => {
  return(
    <img src={LogoImage} alt={alt} className="logo__image" />
  );
};

export default Logo;