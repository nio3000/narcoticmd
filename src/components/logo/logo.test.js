import React from 'react';
import ReactDOM from 'react-dom';
import Logo from './Logo';

import renderer from 'react-test-renderer';


describe( 'Logo', () => {
  it('renders correctly', () => {
    const component = renderer.create(
      <Logo />
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  })
});