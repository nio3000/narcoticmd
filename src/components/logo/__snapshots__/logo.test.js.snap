// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo renders correctly 1`] = `
<img
  alt={Object {}}
  className="logo__image"
  src="head.svg"
/>
`;
